### PR TITLE
change the type

### PR DESCRIPTION
**Pull request by danthegoodman1**
Change-type: minor

### CODE REVIEW

File: balena.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+data:
+  $transformer:
+    dummy: 1
+  platforms: # this field means "I want to be built for these"
+    'linux/amd64': {}
+    # 'linux/arm64': {} # disabled until multi-arch is working without manual changes
+  fragment:
+    type: service@1.0.0
+    data:
+      fooBar: 43
```
Comment (Hades32) [inline balena.yml:13]: looks like c&p

File: balena.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+type: type-product-os-t-node-module-source@0.0.3
+version: 1.0.12
+name: Transformer Runtime Package
+data:
+  $transformer:
+    dummy: 1
+  platforms: # this field means "I want to be built for these"
+    'linux/amd64': {}
+    # 'linux/arm64': {} # disabled until multi-arch is working without manual changes
```
Comment (Hades32) [inline balena.yml:9]: do you plan on using this?

File: balena.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+type: type-product-os-t-node-module-source@0.0.3
+version: 1.2.0
+name: Transformer Runtime Package
+data:
+  $transformer:
+    dummy: 1
```
Comment (Hades32) [inline balena.yml:6]:
```suggestion
  $transformer: {}
```
better so "dummy" doesn't get passed through *all* contracts

**Comment by danthegoodman1** [comment]
@balena-ci I self certify!

**Comment by danthegoodman1** [comment]
oops accidentally told versionbot to approve

**Pull request closed without merging**